components/algolia/actions/create-objects/create-objects.mjs

```diff
@@ -0,0 +1,42 @@
+import algolia from "../../algolia.app.mjs";
+
+export default {
+  key: "algolia-create-objects",
+  name: "Create Objects",
+  description: "Adds an array of JavaScript objects to the given index. [See docs here](https://www.algolia.com/doc/api-reference/api-methods/save-objects/)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    algolia,
+    indexName: {
+      propDefinition: [
+        algolia,
+        "indexName",
+      ],
+    },
+    objects: {
+      label: "Objects",
+      description: "An array of JavaScript objects, each corresponding to a new object in your search index. It's recommended you create this array of objects in a prior code step, then select expression mode and enter a variable reference to that array here",
+      type: "string",
```

Comment:
Optionally, can we use `string[]` type for `objects` prop? It will reduce one level of depth and might be better for user experience.